pkg/bundle/sync.go

```diff
@@ -54,6 +55,11 @@ func (b *bundle) buildSourceBundle(ctx context.Context, bundle *trustapi.Bundle)
 
 		case source.InLine != nil:
 			sourceData = *source.InLine
+
+		case source.CCADB != nil:
+			if *source.CCADB {
+				sourceData = embedded.MozillaCACertificatesPEM()
+			}
```

Comment:
I hadn't really considered the idea of providing a bundle embedded into the trust project itself - it makes sense, especially in airgapped environments.

(The thing I'd envisioned in #8  - which I'd been meaning to write a PR for - was that trust would fetch + use the latest trust bundle available on a regular basis; obviously that's a more complicated thing to do, and doesn't really have to impact this PR)

Reply:
Embedding a set of CAs into this project is something it's not been used for before (up until now it's only ever dealt with moving CAs around, now it is actual a source of CAs). So I think we should consider very carefully whether this is what we want to do (in this way)